ci/prow/fcos-e2e.sh

```diff
@@ -8,4 +8,4 @@ cd $(mktemp -d)
 cosa init --force https://github.com/coreos/fedora-coreos-config/
 cosa fetch
 cosa build
-cosa kola run 'ext.*'
+cosa kola run 'ext.rpm-ostree.*'
```

Comment:
If this is intended to backup CoreOS CI, why not just keep running all the external tests? (Or really just `cosa kola run`.)

Reply:
Well, what tests we run is going to be a long running topic and we need to grow more sophistication here.  Specifically, one thing we can do for example is set up `/test e2e-fcos-all` that runs all the kola tests.

To start I guess my argument is that since we're running more kola tests in the CCI, we don't have a strong need to duplicate them here.  But if CCI is down, then we can tune this up to do more trivially.

Reply:
To rephrase this, the immediate main problem is that our CCI job has a (relatively) high flake rate, so I always need to dig into the logs to see whether e.g. it was the root-raid one, or a compose timeout etc.  Whereas if this current e2e-fcos fails it's so far usually been a real bug in the PR.

Now, we need those compose tests and those extra kola tests; I'm not arguing for not fixing them.  Just that it's best if we have some nuance beyond "run all the tests".